x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/rule_definition_section.tsx

```diff
@@ -341,10 +341,11 @@ const ThreatMapping = ({ threatMapping }: ThreatMappingProps) => {
 
 interface AlertSuppressionTitleProps {
   title: string;
+  ruleType: Type | undefined;
 }
 
-const AlertSuppressionTitle = ({ title }: AlertSuppressionTitleProps) => {
-  return <AlertSuppressionTechnicalPreviewBadge label={title} />;
+const AlertSuppressionTitle = ({ title, ruleType }: AlertSuppressionTitleProps) => {
```

Comment:
nit: Looks like `AlertSuppressionTitle` and `AlertSuppressionLabel` is basically the same component. We might want to remove `AlertSuppressionTitle` entirely and just use `AlertSuppressionLabel` directly.